api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -80,7 +81,12 @@ def invite_member
         status: 'staged' # Will be updated to 'accepted' for unconfirmed users
       )
 
-      @user = user.confirmed? ? invite : user.reload
+      @user = if @new_user
```

Comment:
Set that up since I was initially trying to prevent the double query in `create_unconfirmed_user` method. However, that would require some depper changes that might also need reflection in `maestrano-hub`. Removing this and using `user.confirmed?`